clang/lib/Sema/SemaCodeComplete.cpp

```diff
@@ -197,6 +197,9 @@ class ResultBuilder {
   /// Whether the \p ObjectTypeQualifiers field is active.
   bool HasObjectTypeQualifiers;
 
+  // Whether the member function is using an explicit object parameter
+  bool IsExplicitObjectMemberFunction;
```

Comment:
Happy to rename this, wasn't sure of a good name, especially since filters use the `IsXxx` format.